docs/getting-started-guides/clc.md

```diff
@@ -25,7 +36,7 @@ If you run into any problems or want help with anything, we are here to help. Re
 
 The requirements to run this script are:
 - A linux administrative host (tested on ubuntu and OSX)
-- python 2 (tested on 2.7.11)
+- python 2 (tested on 2.7.11), including the -dev and -crypto packages
   - pip (installed with python as of 2.7.9)
 - git
```

Comment:
version? I'm assuming git is only needed to clone the ansible repo but I'm not sure. If that's the case then git isn't actually needed because a user can download/extract the tar/zip from github.